kitsune/settings.py

```diff
@@ -527,7 +527,26 @@ def path(*parts):
 )
 if READ_ONLY:
     AUTHENTICATION_BACKENDS = ('kitsune.sumo.readonlyauth.ReadOnlyBackend',)
-
+    OIDC_ENABLE = False
+    ENABLE_ADMIN = False
+else:
+    OIDC_ENABLE = config('OIDC_ENABLE', default=False, cast=bool)
+    ENABLE_ADMIN = config('ENABLE_ADMIN', default=OIDC_ENABLE, cast=bool)
+    if OIDC_ENABLE:
+        AUTHENTICATION_BACKENDS = (
+            'mozilla_django_oidc.auth.OIDCAuthenticationBackend',
+        )
+        OIDC_OP_AUTHORIZATION_ENDPOINT = config('OIDC_OP_AUTHORIZATION_ENDPOINT')
+        OIDC_OP_TOKEN_ENDPOINT = config('OIDC_OP_TOKEN_ENDPOINT')
+        OIDC_OP_USER_ENDPOINT = config('OIDC_OP_USER_ENDPOINT')
+
+        OIDC_RP_CLIENT_ID = config('OIDC_RP_CLIENT_ID')
+        OIDC_RP_CLIENT_SECRET = config('OIDC_RP_CLIENT_SECRET')
+        OIDC_CREATE_USER = config('OIDC_CREATE_USER', default=False, cast=bool)
+        MIDDLEWARE_CLASSES = (
```

Comment:
Just a small nit, I think it can be written like `MIDDLEWARE_CLASSES += ('mozilla_django_oidc.middleware.RefreshIDToken',)`